examples/shell/shell_common/cmd_ping.cpp

```diff
@@ -268,17 +268,17 @@ CHIP_ERROR EstablishSecureSession(streamer_t * stream, const Transport::PeerAddr
 
 void HandleEchoResponseReceived(Messaging::ExchangeContext * ec, System::PacketBufferHandle && payload)
 {
-    uint64_t respTime    = System::SystemClock().GetMonotonicMilliseconds();
-    uint64_t transitTime = respTime - gPingArguments.GetLastEchoTime();
-    streamer_t * sout    = streamer_get();
+    System::Clock::Timestamp respTime         = System::SystemClock().GetMonotonicTimestamp();
+    System::Clock::Milliseconds64 transitTime = respTime - gPingArguments.GetLastEchoTime();
```

Comment:
nit: it does not seem obvious that `timestamp - timestamp` returns milliseconds. It may be more obvious if it would return a `TimeDelta` that has toMilliseconds() or toSeconds() or things like that.

For future PRs maybe.

Reply:
> nit: it does not seem obvious that `timestamp - timestamp` returns milliseconds. 

`timestamp - timestamp` returns a `timestamp` which is converted on assignment to milliseconds. Maybe that's too subtle and would be clearer with a `static_cast<Milliseconds64>(timestamp - timestamp)` or something like that?

Reply:
ah, I see Milliseconds64 is a chrono::duration. I read it as a typedef to some uint64_t hence my complain. However if the code is unit-safe, that works for me.